Reports/e2e/e2e-results.trx

```diff
@@ -0,0 +1,34 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<TestRun id="5974b761-41fb-422c-bc49-f18482b2ef74" name="@2a427376d2b7 2025-09-28 12:11:55" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
```

Comment:
Woops, we don't want anything in the Reports directory committed to source control. Can you delete this file from git, and add the Reports directory to the .gitignore?